src/OceanSimulations/prescribed_ocean_model.jl

```diff
@@ -0,0 +1,96 @@
+using Oceananigans.Models: AbstractModel
+using Oceananigans.OutputReaders: extract_field_time_series, update_field_time_series!
+
+import Oceananigans.TimeSteppers: time_step!, update_state!, reset!, tick!
+import Oceananigans.Models: timestepper, update_model_field_time_series!
+
+import ClimaOcean: reference_density, heat_capacity
+import Oceananigans.Architectures: on_architecture
+
+#####
+##### A prescribed ocean...
+#####
+
+struct PrescribedOceanModel{G, C, U, T, F, Arch} <: AbstractModel{Nothing, Arch}
+    architecture :: Arch      
+    grid :: G        
+    clock :: Clock{C}
+    velocities :: U
+    tracers :: T
+    timeseries :: F
+end
+
+"""
+    PrescribedOceanModel(timeseries=NamedTuple(); grid, clock=Clock{Float64}(time = 0))
+
+Create a prescribed ocean model to be used in combination with ClimaOcean's `OceanSeaIceModel` 
+on a `grid` with a `clock`.
+
+Arguments
+=========
+- `timeseries`: A `NamedTuple` containing time series data for various fields. The named tuple can be empty 
```

Comment:
the name is confusing because when we say "timeseries" it means just one series. But this is many timeseries?